roles/common/README.md

```diff
@@ -0,0 +1,77 @@
+common
+======
+
+The tests in this role are not specific to any one functional area but are an
+aggregate of common tests that can be used in all OSP 18.0/OCP jobs.
+
+The available tasks tests are:
+
+* pod tests
+
+Requirements
+------------
+
+The requirements vary according to the tests run.
+
+For the pod tests, access to a kubernetes cluster and the oc command is needed,
+this is done by passing the KUBECONFIG env var and PATH into the play.
+
+The following tools are also needed:
+* grep
+* awk
+
+Role Variables
+--------------
+Variable required for all tasks to run
+
+For pod_tests.yml tasks:
+
+    common_pod_test_id
```

Comment:
Maybe add a description of this variable too? Is this the ID of a test in polarion?

Reply:
Yes, this is a polarion ID number.